items.json

```diff
@@ -4429,9 +4429,9 @@
     "name": "Highland Spiked Club",
     "culture": "Battania",
     "type": "OneHandedWeapon",
-    "price": 107,
-    "weight": 1.42,
-    "tier": 0.355965823,
+    "price": 949,
+    "weight": 1.46,
+    "tier": 2.86432219,

```

Comment:
maybe this one should be higher tier as it looks not too bad? 
![image](https://user-images.githubusercontent.com/47090987/202019577-a50c5455-1008-4a0f-82b0-5bc9c5dde62f.png)